src/endpoints/sources.js

```diff
@@ -352,6 +352,11 @@ function receiveSubresource(subresourceExtractor) {
 
     authz.assertAuthorized(req.user, authz.actions.Write, subresource.resource);
 
+    /* Notify the client they can continue sending the request body now that
+     * we're past authz.
+     */
+    if (req.expectsContinue) res.writeContinue();
```

Comment:
Is `100-Continue` only used for `PUT` requests in this app? If so, any chance it'll be used elsewhere?

Reply:
It'd be used for any body-bearing methods, like PUT, POST, or PATCH, but this endpoint is the only one we currently have.